BusinessCore.Tests/FunctionalProgramingTests/SodenFunctionalProgramingTests.cs

```diff
@@ -19,7 +22,9 @@ public SodenFunctionalProgramingTests()
         }
 
         [TestMethod]
-        public void a_there_are_3_black_vehicles_and_horsePorwer_min100_and_started()
+        [TestCategory("Soden")]
```

Comment:
If you add categorioes at class level, you do not need categories at method level.
Please rename method category.